tests/TestCase/Http/Middleware/CsrfProtectionMiddlewareTest.php

```diff
@@ -309,4 +310,24 @@ public function testConfigurationValidate()
         $response = $middleware($request, $response, $this->_getNextClosure());
         $this->assertInstanceOf(Response::class, $response);
     }
+
+    /**
+     * @return void
+     * @doesNotPerformAssertions
+     */
+    public function testFoo()
+    {
+        $request = new ServerRequest([
+            'environment' => [
+                'REQUEST_METHOD' => 'POST',
+                'ALLOW_ME' => 'let-me-pass'
+            ],
+        ]);
+        $response = new Response();
+
+        $middleware = new CsrfProtectionMiddleware();
+        $middleware->whitelistCallback(function (ServerRequestInterface $request): ?bool {
```

Comment:
No return types for PHP5 :)

Reply:
Doh, I should have made PR for 4.x and let someone else backport it :P